src/lib/components/Charts/ColumnChart.jsx

```diff
@@ -88,7 +92,7 @@ class ColumnChart extends React.Component {
                 return `<span>${this.y}</span>`;
               }
             },
-          },
+          },*/
```

Comment:
Todo - Make these tooltip callbacks optional and extensible through props (not necessary for `v1.1.0` I don't think)